controls/busyindicator/integrate-with-listview.md

```diff
@@ -1,68 +0,0 @@
----
-title: Integration
-page_title: .NET MAUI BusyIndicator Documentation - ListView Integration
-description: Learn more about the option to integrate BusyIndacator with ListView control.
-position: 2
-previous_url: /controls/busyindicator/busyindicator-integrate-with-listview
-slug: busyindicator-integrate-with-listview
```

Comment:
Because this page is deleted, slug-based links to it no longer work. Please, check these error messages and fix the links on the respective pages as needed.

```
ERROR: No page with slug `busyindicator-integrate-with-listview` in /controls/busyindicator/content. Consider fixing the slug or use normal link.
             ERROR: No page with slug `busyindicator-integrate-with-listview` in /controls/busyindicator/getting-started. Consider fixing the slug or use normal link.
```